clang/lib/Sema/SemaChecking.cpp

```diff
@@ -7513,6 +7516,41 @@ void Sema::checkCall(NamedDecl *FDecl, const FunctionProtoType *Proto,
       }
     }
 
+    auto *CallerFD = dyn_cast<FunctionDecl>(CurContext);
+    bool IsCalleeStreaming = ((ExtInfo.AArch64SMEAttributes &
+                               FunctionType::SME_PStateSMEnabledMask) ||
+                              (ExtInfo.AArch64SMEAttributes &
+                               FunctionType::SME_PStateSMCompatibleMask));
```

Comment:
Streaming-compatible and streaming are not the same thing. i.e. if the callee is 'streaming-compatible', `IsCalleeStreaming` should be `false`.

Reply:
Done.